lib/internal/Magento/Framework/Filesystem/Io/Sftp.php

```diff
@@ -119,18 +121,17 @@ public function rmdir($dir, $recursive = false)
                     if ($this->_connection->chdir($filename)) {
                         // This is a directory
                         $this->_connection->chdir('..');
-                        $no_errors = $no_errors && $this->rmdir($filename, $recursive);
+                        $noErrors = $noErrors && $this->rmdir($filename, $recursive);
                     } else {
-                        $no_errors = $no_errors && $this->rm($filename);
+                        $noErrors = $noErrors && $this->rm($filename);
                     }
                 }
             }
-            $no_errors = $no_errors && ($this->_connection->chdir(
+            return $noErrors && ($this->_connection->chdir(
```

Comment:
I would recommend leaving it as it was. Having a complex expression in the return statement may be not the best

Reply:
Done!